### ISSUE TITLE
Add Yarn to necessary install instructions. 

### ISSUE THREAD

**Issue by julienlegault** [nminchow/Artifinder#4]
Yarn is needed to run this app. Please add a link to https://classic.yarnpkg.com/en/ in the Readme.md file.

**Comment by nminchow** [comment]
@julienlegault - are you running it with the container? It looks like the build wasn't actually installing the dependencies as intended 😬 - it should be fixed by cbaa15aedd2a2ad52002a0b6f699d98eecfeb47e, if you don't mind giving it another shot.

**Comment by julienlegault** [comment]
PS C:\Users\aquin\Documents\Code\Artifinder> docker-compose up
Starting artifinder_artifinder_1 ... done
Attaching to artifinder_artifinder_1
artifinder_1  | yarn run v1.22.4
artifinder_1  | $ vue-cli-service serve
artifinder_1  | /bin/sh: 1: vue-cli-service: not found
artifinder_1  | error Command failed with exit code 127.
artifinder_1  | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
artifinder_artifinder_1 exited with code 127

There is my full error code now. I'm wondering if it's because it's being set to production environment? Or I just have far too many dependencies that are out of date now. Either way, it may wind up easier to start from scratch.

**Comment by nminchow** [comment]
@julienlegault - did you pull latest master?

**Comment by julienlegault** [comment]
Yes, but I've decided to start from scratch anyways. Docker is a ridiculous mess that I don't want to touch. I'm not really sure why you used as many extra tools as you did for this small of a project... All of them seem unnecessary.

**Comment by nminchow** [comment]
I have found docker has eased my workflow quite a bit as I develop across 2 OSs. 

I just cloned on a fresh machine without updating any dependencies and the app started fine, so I don't believe it is failing to build due to outdated dependencies. That said - I haven't been maintaining this project actively, so starting from a different base isn't a terrible idea. Regardless, best of luck on your project!
